go-selinux/selinux_linux.go

```diff
@@ -282,6 +282,23 @@ func readCon(fpath string) (string, error) {
 	return strings.Trim(retval, "\x00"), nil
 }
 
+func ClassIndex(class string) int {
```

Comment:
Please add documentation

Reply:
The usual go convention is to return a value and an error. Returning -1 as an indication of an error is way too C-like, and we lose the error information.